projects/utilities/pathfinder_cli/main.cpp

```diff
@@ -204,10 +206,29 @@ int main(int argc, char const *argv[]) {
       //TODO() Enable Inputs to be in degrees West/South
       auto points = vm["navigate"].as<std::vector<double>>();
 
-      start_lat = (points[0]);
-      start_lon = (points[1]);
-      end_lat = (points[2]);
-      end_lon = (points[3]);
+      NetworkTable::NonProtoConnection connection;
+      if (vm.count("t")) {
+        // Connect to the network table
+        try {
+            std::cout << "Connecting to network table" <<std::endl;
+            connection.Connect(100);
+        } catch (NetworkTable::TimeoutException) {
+            std::cout << "Failed to connect" << std::endl;
+            return 0;
+        }
+
+        std::pair<double, double> gps_coords;
+        gps_coords = connection.GetCurrentGpsCoords();
+        start_lat = (int) gps_coords.first;
```

Comment:
I see the coords.first is lat and coords.second is lon here. Just want to confirm if that is correct, since the other code has it the other way around.

Reply:
This is correct as GetCurrentGpsCoords() returns the coordinates as (lat, lon).

But if it's decided that we should follow a (lon, lat) convention, we can make updates on NAV side